### ISSUE TITLE
Pass Rails secret environment variable to Cypress

### ISSUE THREAD

**Issue by jonnovaretti** [shakacode/cypress-on-rails#121]
I manage my secret keys in the `config/credentials.yml` as normal Rails app.

I'd like to keep all secrets in the credentials file therefore I want to add my keys there and use them in Cypress e2e specs.

Is that possible?

**Comment by grantspeelman** [comment]
yes, you can use an APP Command https://github.com/shakacode/cypress-on-rails#example-of-using-app-commands
the last thing in a command is returned to cypress.

Example of using a returned value

```js
cy.app('load_credentials').then((response) => {
      // use response in then
      cy.type(response.password)
 })
```

**Comment by jonnovaretti** [comment]
@grantspeelman, it works perfectly, thanks!
